debian/ceph-mgr-dashboard.install

```diff
@@ -1 +1,7 @@
-usr/share/ceph/mgr/dashboard
+usr/share/ceph/mgr/dashboard/*.py
+usr/share/ceph/mgr/dashboard/README.rst
+usr/share/ceph/mgr/dashboard/controllers/*.py
+usr/share/ceph/mgr/dashboard/plugins/*.py
+usr/share/ceph/mgr/dashboard/services/*.py
+usr/share/ceph/mgr/dashboard/frontend/package*.json
```

Comment:
why do we need to install package*.json, i think they describe the node package dependencies. presumably, they are already packaged in this change, right?

Reply:
Yeah, but it's also used in [Python for guessing the default language](https://github.com/ceph/ceph/pull/30694/files#diff-acfcd85243015893c54b6337fe72c29dR31). We could look for an alternative approach (ini file, ceph OPTION, ...), but we wanted to limit the number of sources of truth to 1.

On the `package-lock.json`, while not necessary, it gives a clue on the versions of packages used for build the front-end. Now I think about it, we would probably want to generate and save sourcemaps instead for debugging minified Javascript.

Reply:
> Yeah, but it's also used in [Python for guessing the default language](https://github.com/ceph/ceph/pull/30694/files#diff-acfcd85243015893c54b6337fe72c29dR31). We could look for an alternative approach (ini file, ceph OPTION, ...), but we wanted to limit the number of sources of truth to 1.
> 

i see, could you s/package*.json/package.json/ ?

> On the `package-lock.json`, while not necessary, it gives a clue on the versions of packages used for build the front-end. Now I think about it, we would probably want to generate and save sourcemaps instead for debugging minified Javascript.

it's like shipping debugsymbols to user even though user is not interested in them. i think each source tarball is versioned. and the build process is repeatable. probably we don't need to worry about if the version of a certain node package cannot be tracked?

also, i am not quite sure if "dh_install" will install all "package*.json" when matching "usr/share/ceph/mgr/dashboard/frontend/package*.json". presumably the wildcard pattern should be `"usr/share/ceph/mgr/dashboard/frontend/node_modules/*/package*.json"` ?

Reply:
I'm trying with a different approach (filtering them in the cmake install process, as it was also causing a issue with `rpmbuild` post checks). I'm doing now a gentle filtering just to check it works as expected and later will narrow it down a bit.